### PR TITLE
Delete meetings

### PR DESCRIPTION
**Pull request by juzzztinsoong**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.meeting.Meeting;
+
+/**
+ * Adds a meeting to the address book.
```
Comment (howenc) [inline src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java:15]: Deletes a meeting to the address book.

File: src/main/java/seedu/address/model/meeting/TitleContainsKeywordsPredicate.java
Comment (howenc) [inline src/main/java/seedu/address/model/meeting/TitleContainsKeywordsPredicate.java:1]: What does this class do?

**Review by howenc — APPROVED** [review]
Sync and push again, should be alright.

File: src/main/java/seedu/address/model/meeting/TitleContainsKeywordsPredicate.java
Comment (juzzztinsoong) [reply src/main/java/seedu/address/model/meeting/TitleContainsKeywordsPredicate.java:1]: implemented if we want to search for meetings by keyword/name

**Review by howenc — APPROVED** [review]

**Pull request merged**